### DISCUSSION TITLE
TextField values not showing when passed to another function

### DISCUSSION THREAD

**Discussion by Haadi-A** [flet-dev/flet#907] [Q&A] [answered]
I have TextField which take data and gives it to another function but the problem is when I pass the value of the TextField like txtfield.value to another function it shows the flet TextField object's location in memory like this: 
 <flet.textfield.TextField object at 0x7ff0e8e644f0> 
instead of the value of the text field is there any way to get the values in the other function?

code structure is roughly like this:

        def main(page:ft.Page):
                 txtFieldobj = ft.TextField(
                 autofocus=True, label="Student's full name", width=250)

                 button = ft.Column(controls=[
                                ft.ElevatedButton(
                                    text="Done", width=100, on_click=onBttonClicked),
                            ], )
             def onButtonClicked(e):
                   theValue = txtFieldObj.value
                  sendDataToDatabase(theValue)
                  print(theValue)  # -> shows the object's address instead of the object's value <flet.textfield.TextField object at 0x7ff0e8e644f0>
    
            def sendDataToDatabase(theValue):
                  useTheValue(theValue)
             
            page.add(
                    ft.Column(controls = [ txtFieldObj, button,]
                 )  
             )

**Comment by ndonkoHenri** [comment, marked as answer]
Hey, grab the code below which works:
```python
import flet as ft


def main(page: ft.Page):
    def onButtonClicked(e):
        print(txtFieldobj.value)

    txtFieldobj = ft.TextField(
        autofocus=True, label="Student's full name", width=250)

    button = ft.Column(controls=[
        ft.ElevatedButton(
            text="Done", width=100, on_click=onButtonClicked),
    ], )

    page.add(
        ft.Column(controls=[txtFieldobj, button, ]
                  )
    )
ft.app(target=main)
```

**Comment by Haadi-A** [comment]
Thanks that helpful.
